src/Uno.Extensions.Authentication.MSAL/build/Package.targets

```
@@ -36,4 +36,16 @@
 		</ItemGroup>
 		<Message Importance="high" Text ="Removed: @(_WebView2CoreOutputsToExclude)" />
 	</Target>
+
+	<Target Name="_ValidateMsalDependencyWasm"
+			BeforeTargets="Compile"
+			Condition="'$(IsUnoHead)'=='true' and '$(UnoRuntimeIdentifier)'=='WebAssembly'">
+		<ItemGroup>
+			<_ClientRefs Include="@(ReferencePath)" Condition="'%(ReferencePath.NuGetPackageId)' == 'Microsoft.Identity.Client'" />
+			<_MsalRefs Include="@(ReferencePath)" Condition="'%(ReferencePath.NuGetPackageId)' == 'Microsoft.Identity.Client.Extensions.Msal'" />
+		</ItemGroup>
+		<Error Condition="'@(_ClientRefs)'==''" Text="In order to use Uno MSAL support, the 'Microsoft.Identity.Client' NuGet Package must be referenced in the project. You can add &lt;PackageReference Include=&quot;Microsoft.Identity.Client&quot; Version=&quot;4.61.3&quot; /&gt; to your project." />
```

```suggestion
		<Error Condition="'@(_ClientRefs)'==''" Text="In order to use Uno.Extensions.Authentication.MSAL, the 'Microsoft.Identity.Client' NuGet Package must be referenced in the project. You can add &lt;PackageReference Include=&quot;Microsoft.Identity.Client&quot; Version=&quot;4.61.3&quot; /&gt; to your project." />
```

Let's avoid providing a specific version since this will get out of date.